src/content/docs/en/core-concepts/astro-syntax.mdx

```diff
@@ -134,22 +134,7 @@ An Astro component template can render multiple elements with no need to wrap ev
 <p>Astro supports multiple root elements in a template.</p>
 ```
 
-However, when using an expression to dynamically create multiple elements, you should wrap these elements inside a **fragment** as you would in JavaScript or JSX. Astro supports using either `<Fragment> </Fragment>` or the shorthand `<> </>`.
-
-```astro title="src/components/FragmentWrapper.astro" "<>" "</>"
----
-const items = ["Dog", "Cat", "Platypus"];
----
-<ul>
-  {items.map((item) => (
-    <>
-      <li>Red {item}</li>
-      <li>Blue {item}</li>
-      <li>Green {item}</li>
-    </>
-  ))}
-</ul>
-```
+Astro supports using either `<Fragment> </Fragment>` or the shorthand `<> </>`.
```

Comment:
```suggestion
However, you can use fragments to ... Astro supports using either `<Fragment> </Fragment>` or the shorthand `<> </>`.
```
This sentence on its own feels a little like, "Wait, you just said I don't need fragments?!"  :smile:

I think one use of Fragment we show elsewhere is avoiding extra wrapping HTML elements, especially when using `set:html`. What about adding something here so that this section says both that we DO support them, and when you might use them?